js_modules/dagster-ui/packages/ui-core/src/overview/OverviewAssetsRoot.tsx

```diff
@@ -44,7 +44,7 @@ export const OverviewAssetsRoot = ({Header, TabButton}: Props) => {
   useDocumentTitle('Overview | Assets');
 
   const {assets, query, error, loading} = useAllAssets();
-  const refreshState = useRefreshAtInterval({
+  const refreshState = useRefreshAtInterval<any>({
```

Comment:
Was this necessary? Doesn't seem related to other stuff in this PR

Reply:
yeah it is, switching Workspace to useIndexedDBCachedQuery revealed some weirdness in the typing

Reply:
> Managing a stack is hard

graphite cli is worth trying if you haven't